optuna/integration/wandb.py

```diff
@@ -212,6 +189,33 @@ def track_in_wandb(self) -> Callable:
         All the metrics from inside the objective function will be logged into the same run
         which stores the parameters for a given trial.
 
+        Example:
+
+            Add additional logging to Weights & Biases.
+
+            .. code::
+
+                import optuna
+                from optuna.integration.wandb import WeightsAndBiasesCallback
+                import wandb
+
+                wandb_kwargs = {"project": "my-project"}
+                wandbc = WeightsAndBiasesCallback(wandb_kwargs=wandb_kwargs)
+
+
+                @wandbc.track_in_wandb()
+                def objective(trial):
+                    x = trial.suggest_float("x", -10, 10)
+                    mlflow.log_param("power", 2)
+                    mlflow.log_metric("base of metric", x - 2)
```

Comment:
Sorry I cannot test this code because I am not a user of wandb, but is it possible to use `mlflow.log_param` or `mlflow.log_metric` in `wandbc.track_in_wandb`? If so, doesn't this example need `import mlflow`?

Reply:
Good catch! I guess the `mlflow` example happened to contaminate the wandb example.
We can simply move the example in this file.
```
            @wandbc.track_in_wandb()
            def objective(trial):
                x = trial.suggest_float("x", -10, 10)
                loss = (x - 2) ** 2
                wandb.log({"loss": loss})
                return loss
```

Reply:
Thank you for pointing out this! I intended to use a similar example code to mlflow because the original example was not informative; `loss` is stored as an objective value and `loss` is overwritten by each`wandb.log` with WeightsAndBiasesCallback(multi_run=False), which is the default.


I revised the example as in https://github.com/optuna/optuna/pull/3477/commits/b9a0f576975d53e0ff40787bc9900248f36954ef.

Reply:
Thanks, LGTM!